usr/lib/tik/modules/pre/15-encrypt

```diff
@@ -61,7 +61,7 @@ encrypt_notification() {
         # Secureboot being disabled makes the notification a Yes/No with the preference being to exit
         if [ "${secureboot_disabled}" == 1 ]; then
             log "[encrypt_notification] secureboot warning shown"
-            zenity --width=600 --question --icon=security-low-symbolic --title="Warning" --ok-label="Cancel Installation" --cancel-label="I Understand, Proceed Anyway" --text="${preamble}\n\nReason: <b>SecureBoot Disabled</b> and ${reason}\n\n${secureboot_warning}" && exit 1
+            d --width=600 --question --icon=security-low-symbolic --title="Warning" --ok-label="Cancel Installation" --cancel-label="I Understand, Proceed Anyway" --text="${preamble}\n\nReason: <b>SecureBoot Disabled</b> and ${reason}\n\n${secureboot_warning}" && exit 1
```

Comment:
This will not work. The reason it's structured like this is so the zenity's right most (default) button will be 'Cancel Installation' and the left most (normally cancel) button will be 'I Understand, Proceed Anyway.

`d` will interpret the 'Cancel' as a failure, as abort tik. Whereas the `&& exit 1` will also exit tik..so this is a functional dead end :) 

I guess we need an some kind of flag/variant of `d` that removes the return code checking so we can preserve this behaviour while working better with cenity (love the name btw)

Maybe `d-opt` the same way we have `prun-opt` which does the same for `prun` as we now need for `d`

Dont' think this problem is serious enough to block merging, but will need to be addressed before the next tik release.